react/features/conference/components/native/carmode/CarMode.tsx

```diff
@@ -26,16 +33,40 @@ import TitleBar from './TitleBar';
 // @ts-ignore
 import styles from './styles';
 
+
 /**
  * Implements the carmode component.
  *
  * @returns { JSX.Element} - The carmode component.
  */
-const CarMode = (): JSX.Element => {
+const CarMode = ({ navigation }: any): JSX.Element => {
```

Comment:
I don't think we need any of these changes anymore.

Reply:
Reverted.